lnwallet/interface.go

```diff
@@ -84,6 +84,31 @@ type TransactionDetail struct {
 	DestAddresses []btcutil.Address
 }
 
+type byTimestamp []TransactionDetail
+
+func (t byTimestamp) Len() int {
```

Comment:
Similar comment here: we can avoid the new struct by using: https://golang.org/pkg/sort/#Slice

Reply:
modified